src/util.h

```diff
@@ -175,4 +175,12 @@ static SECP256K1_INLINE void memczero(void *s, size_t len, int flag) {
     }
 }
 
+/** If flag is true, set *r equal to *a; otherwise leave it. Constant-time.  Both *r and *a must be initialized.*/
+static SECP256K1_INLINE void secp256k1_int_cmov(int *r, const int *a, int flag) {
+    unsigned int mask0, mask1;
+    mask0 = flag + ~0u;
+    mask1 = ~mask0;
+    *r = (*r & mask0) | (*a & mask1);
+}
```

Comment:
I'm sure people won't like this but this is perfectly well-defined. :p 
```suggestion
static SECP256K1_INLINE void secp256k1_int_cmov(int *r, const int *a, int flag) {
    int mask0, mask1;
    mask0 = 1 - flag;
    mask1 = flag;
    *r = (*r * mask0) + (*a * mask1);
}
```

Reply:
Nice!
Interesting to think of a cmov using arithmetic and not bitwise masking.
Why won't people like it?

Reply:
> Interesting to think of a cmov using arithmetic and not bitwise masking.

You see that often in pseudocode.

> Why won't people like it?

Integer multiplication is slower than `&`. But psst, let's not start the discussion and go with the `VERIFY`. ;)